### PR TITLE
test(js): Convert ProjectKeys to RTL

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/views/settings/project/projectKeys/list/index.spec.jsx [hunk 1]
```diff
@@ -47,39 +33,64 @@ describe('ProjectKeys', function () {
...
 
-    wrapper = mountWithTheme(
+    render(
       <ProjectKeys routes={[]} params={{orgId: org.slug, projectId: project.slug}} />
     );
 
-    expect(wrapper.find('EmptyMessage')).toHaveLength(1);
+    expect(screen.getByTestId('empty-message')).toHaveTextContent(
+      'There are no keys active for this project.'
+    );
```
Comment (priscilawebdev) [inline static/app/views/settings/project/projectKeys/list/index.spec.jsx:42]:
```suggestion
   expect(screen.getByText('There are no keys active for this project.')).toBeInTheDocument()
```

File: static/app/views/settings/project/projectKeys/list/index.spec.jsx [hunk 2]
```diff
@@ -47,39 +33,64 @@ describe('ProjectKeys', function () {
...
+        routes={[]}
+        params={{orgId: org.slug, projectId: project.slug}}
+        project={TestStubs.Project()}
+      />
+    );
+
+    const expandButton = screen.getByRole('button', {name: 'Expand'});
+
+    expect(expandButton).toBeInTheDocument();
+    userEvent.click(expandButton);
```
Comment (priscilawebdev) [inline static/app/views/settings/project/projectKeys/list/index.spec.jsx:57]:
```suggestion
    userEvent.click(screen.getByRole('button', {name: 'Expand'}));
```

File: static/app/views/settings/project/projectKeys/list/index.spec.jsx [hunk 3]
```diff
@@ -47,39 +33,64 @@ describe('ProjectKeys', function () {
...
+
+    expect(expandButton).not.toBeInTheDocument();
```
Comment (priscilawebdev) [inline static/app/views/settings/project/projectKeys/list/index.spec.jsx:59]:
```suggestion
    expect(screen.queryByRole('button', {name: 'Expand'})).not.toBeInTheDocument();
```

File: static/app/views/settings/project/projectKeys/list/index.spec.jsx [hunk 4]
```diff
@@ -47,39 +33,64 @@ describe('ProjectKeys', function () {
...
+    render(
+      <ProjectKeys
+        routes={[]}
+        params={{orgId: org.slug, projectId: project.slug}}
+        project={TestStubs.Project()}
+      />
+    );
+
+    userEvent.click(screen.getByRole('button', {name: 'Delete'}));
+    render(<GlobalModal />);
```
Comment (priscilawebdev) [inline static/app/views/settings/project/projectKeys/list/index.spec.jsx:72]: interesting🤔

**Comment by evanpurkhiser** [comment]
Sure, yeah I can agree with your thoughts here Pri, let me adjust!

**Review by scttcper — APPROVED** [review]

**Pull request merged**